src/overlays/actors/ovl_Obj_Chan/z_obj_chan.c

```diff
@@ -119,7 +119,7 @@ u32 func_80BB9A1C(ObjChan* this, f32 arg1) {
     sp20 = Math_SinS(this->unk1D4) * this->unk1D0;
     temp_f6 = (Math_CosS(this->unk1D4) * 0.03834952f * this->unk1D0) + arg1;
     if (temp_f6 != 0.0f) {
-        this->unk1D4 = RADF_TO_BINANG(func_80086B30(sp20 * 0.03834952f, temp_f6));
+        this->unk1D4 = RAD_TO_BINANG(func_80086B30(sp20 * 0.03834952f, temp_f6));
```

Comment:
This float is `400 * M_PI / 0x8000`, for some reason, so it's probably an angle conversion.